### ISSUE TITLE
Wagtail: Layman request /current-user does not return "username"

### ISSUE THREAD

**Issue by jmacura** [hslayers/hslayers-ng#2923]
# Bug

**Describe the bug**
After using the "Login to catalogue" button in Wagtail-integrated app, after a successful login is done, a modal window pops-up. The "close" link in the modal window won't close it.

**To Reproduce**
1. Go to https://hub4everybody.lesprojekt.cz/map-demo/
2. Click on 'Datasources' and log in to catalogue
3. Click on the "close" text in the modal dialog
4. See nothing is happening

**Expected behavior**
The modal dialog window closed.

**Screenshots**
![image](https://user-images.githubusercontent.com/5598693/164242794-d528284d-805e-4dff-a91a-d2f56c15524b.png)

**Additional context**
none

**Comment by fzadrazil** [comment]
The culprit is probably in the ensureUsername, which should reserver the login in Layman database. That did not happen for a new user login in Wagtail. Should be called automatically after the first "Login to catalogue" event.
